pom.xml

```diff
@@ -561,5 +562,21 @@ limitations under the License.
         <module>fluent-gen</module>
       </modules>
     </profile>
+    <profile>
```

Comment:
I don't know if it is possible in a pom, but is there a way to inherit this? I'd really prefer to avoid duplication if we can avoid it.

Reply:
i'm not sure about that, now it's already declared in the root-level pom. as far as i know, it's not inheriting from any other project/places.